extensions/azure-extensions/src/main/java/io/druid/storage/azure/AzureAccountConfig.java

```diff
@@ -27,23 +27,23 @@
 public class AzureAccountConfig
 {
   @JsonProperty
-  private String protocol = "https";
+  public String protocol = "https";
```

Comment:
Is it possible to leave these private and override the getters if needed in unit tests?